reservation/test_schedule_timings.py

```diff
@@ -0,0 +1,132 @@
+from django.test import TestCase
+from django.urls import reverse
+from django.utils import timezone
+from datetime import timedelta
+from reservation.models import Schedule
+from account.models.professional import Professional, Professions
+from account.models.profile import Profile, UserType
+from reservation.forms import ScheduleForm
+from django.contrib.auth.models import User
+
+current_datetime = timezone.now()
+
+
+class ScheduleTestCase(TestCase):
+    def setUp(self):
```

Comment:
Please use fixtures,
Please remove TestCase and setup, it is part of unittest framework, 
This project use pytest

Use fixtures already existe with make_fixtures you can updae all the data needed while parsing vars to fixtures factories

Reply:
Done